cmd/integration/integration.go

```diff
@@ -764,29 +764,29 @@ func main() {
 	}
 	wg.Wait()
 
-	// Check that kubelet tried to make the pods.
+	// Check that kubelet tried to make the containers.
 	// Using a set to list unique creation attempts. Our fake is
 	// really stupid, so kubelet tries to create these multiple times.
-	createdPods := util.StringSet{}
+	createdConts := util.StringSet{}
 	for _, p := range fakeDocker1.Created {
 		// The last 8 characters are random, so slice them off.
 		if n := len(p); n > 8 {
-			createdPods.Insert(p[:n-8])
+			createdConts.Insert(p[:n-8])
 		}
 	}
 	for _, p := range fakeDocker2.Created {
 		// The last 8 characters are random, so slice them off.
 		if n := len(p); n > 8 {
-			createdPods.Insert(p[:n-8])
+			createdConts.Insert(p[:n-8])
 		}
 	}
 	// We expect 9: 2 pod infra containers + 2 pods from the replication controller +
```

Comment:
Should this comment be updated too? 2 containers from the replication controller pods